fe/src/main/java/org/apache/doris/qe/GlobalVariable.java

```diff
@@ -56,6 +56,10 @@
     @VariableMgr.VarAttr(name = "query_cache_size")
     private static long queryCacheSize = 1048576;
 
+    // the flags passed to Backends
+    @VariableMgr.VarAttr(name = "is_set_default_rowset_to_beta")
+    public static boolean isSetDefaultRowsetToBeta = false;
```

Comment:
Why not use variable like `set default_rowset_type = "alpha[beta]"` ?
If we add another rowset type in future, should we add a new variable named `isSetDefaultRowsetToGamma`?

Reply:
I think in that case, a bit(maybe new or the old) will be used as flag to represent Gamma, add a new variable is reasonable, and more, it is more simple to use boolean as a switch.

Reply:
ok, I will change it to string type